strings/en/mention.json

```
@@ -1,8 +1,4 @@
 {
-	"errorNoPing": "You don't have that language's proofreader role, ",
-	"errorNoPingPr": "so you can't mention it.",
-	"errorNoPingTr": "so you can't mention its translator role.",
-	"errorNoPingAll": "so you can't mention the entire language.",
-	"errorNoPingDisclaimer": "Contact the server owner or administrator if you really need to.",
+	"errorNoPing": "You don't have that language's proofreader role, {role, select, pr {so you can't mention it} tr {so you can't mention its translator role} other {so you can't mention the entire language}}. Contact the server owner or administrator if you really need to.",
```

Please adjust the code accordingly
```suggestion
	"errorNoPing": "You don't have that language's proofreader role, {role, select, pr {so you can't mention it} tr {so you can't mention its translator role} all {so you can't mention the entire language}}. Contact a server administrator if you really need to.",
```

---

There's always should be `other`

---

Edited